dev/custom-pages/azindex.js

```diff
@@ -0,0 +1,82 @@
+(function() {
+
+    var $jumpTo       = $('#jump-to'),
```

Comment:
This can be removed, since we're not using a dropdown for the a-z links anymore; the variable is no longer used